hecdssvue/build.gradle

```diff
@@ -0,0 +1,46 @@
+plugins{
+    id 'library.deps-conventions'
+    id 'library.java-conventions'
+
+    // The following plugin is used to build the OSGI bundle
+    id 'biz.aQute.bnd.builder' version '6.4.0'
+}
+
+jar {
+    archiveBaseName.set('hecdssvue')
+    archiveVersion.set('1.0.0.0')
```

Comment:
Same comments about hard-coded and missing `gov.ca.dwr` prefix

Reply:
Version and naming is presently hard coded to align with the eclipse dropin expections. Versioning will likely be revised under milestone 1.1.5 when the final/complete WRIMS Gui build is developed. Hao requested to set the name of the module as "dwr-hecdssvue"

Reply:
Issue link for tracking the versioning update:
https://github.com/CentralValleyModeling/wrims/issues/149